test/test_configuration.rb

```diff
@@ -1,23 +1,25 @@
-require 'helper'
+# rubocop:disable Metrics/ClassLength
```

Comment:
@pathawks Yep, you can safely exclude all of `test/` from `Metrics/ClassLength`. I still would like them tested for the other cops as they're arguably the worst part of our codebase 😆